ui/src/lib.ts

```diff
@@ -44,11 +44,13 @@ const modalDateFormat: Function = ({
   if (startDate === endDate && allDay) {
     return `${startDate}`;
   } else if (startDate === endDate) {
-    return `${startDate} ${timeFormat(start)} - ${timeFormat(end)}`;
+    return `${startDate} · ${timeFormat(start)} - ${timeFormat(end)}`;
   } else if (allDay) {
     return `${startDate} - ${endDate}`;
   } else {
-    return `${startDate} ${timeFormat(start)} - ${endDate} ${timeFormat(end)}`;
+    return `${startDate}, ${timeFormat(start)} - ${endDate}, ${timeFormat(
```

Comment:
why not make this one a dot too?

Reply:
Similar response to the one I left above.